### DISCUSSION TITLE
How to add license to my project?

### DISCUSSION THREAD

**Discussion by ZedZipDev** [community/community#35299] [New to GitHub] [answered]
I have added new project but by some reasons did not add a license.
https://github.com/ZedZipDev/SQLCLR-SpellChecker

Later I have added the LICENSE file as described there:
https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository

All is ok, I see LICENSE.md file in branch but it did not appear in the project. Also, the project has no "license" in the right side,
How to merge this file with main branch? 
How to add the license?

**Comment by radar-blip** [comment, marked as answer]
Go to the branch that says `add-license-1` and click on:  This branch is [1 commit ahead](https://github.com/ZedZipDev/SQLCLR-SpellChecker/compare/main...add-license-1), [1 commit behind](https://github.com/ZedZipDev/SQLCLR-SpellChecker/compare/add-license-1...main) main.
From there, create a pull request and merge them.
You will be merging add-license-1 into main. And your license should then be in the main branch. 

![addlicense](https://user-images.githubusercontent.com/39196767/194228821-7f2091d6-013b-44c7-b721-58e2f1c73149.png)

**Reply by ZedZipDev** [reply]
Thanx, I did it but how to merge? 
Now I have this:
<img width="931" alt="git_001" src="https://user-images.githubusercontent.com/25686753/194234409-153bac69-605a-4ffd-a9c3-65a92679f8aa.PNG">

**Reply by radar-blip** [reply]
Click the green button 'New pull request'

**Reply by ZedZipDev** [reply]
Ok, done. Now I see
<img width="906" alt="git_002" src="https://user-images.githubusercontent.com/25686753/194235852-6b7d3198-d477-4d51-b8b1-c298b1fa6cf1.PNG">

**Reply by DenverCoder1** [reply]
@ZedZipDev On the page for the pull request, you should see a button near the bottom to merge the pull request

Click the pull request title to get to that page.

![image](https://user-images.githubusercontent.com/20955511/194236339-22770a5d-00f5-47b8-aa6c-1cccb67a6898.png)

**Reply by radar-blip** [reply]
Ok. Change the `compare main` to `add-license-1` and the green button should be clickable.

**Reply by ZedZipDev** [reply]
Sorry, ok, done, I see

<img width="418" alt="git_003" src="https://user-images.githubusercontent.com/25686753/194239391-137f6a54-5ce5-4221-a7fe-3ca025cf820b.PNG">

**Reply by radar-blip** [reply]
Hmm, should be done by now. Did you click the new pull request button?

**Reply by DenverCoder1** [reply]
It looks like you've already created the pull request, so you just need to merge it

You should see a green button that says "Merge Pull Request" or similar on this page: https://github.com/ZedZipDev/SQLCLR-SpellChecker/pull/2

![image](https://user-images.githubusercontent.com/20955511/194241384-1cce6499-b99f-4a2c-b298-aa47d467d839.png)

**Reply by radar-blip** [reply]
Thanks @DenverCoder1 👍🏻

**Reply by ZedZipDev** [reply]
Thank you! Finally I have merged and now the LICENSE file appeared in the projects file list and I see the license on the right side.

**Reply by radar-blip** [reply]
Awesome! I knew you could do it!
Yes, I can see your license there too!

**Reply by DSDmark** [reply]
> If you got your answer then you should close this question by click **accept answer** button  @ZedZipDev

**Reply by ZedZipDev** [reply]
Ok, done

**Reply by casionone** [reply]
Hi @linuxlawson about [Apache-2.0, Unknown licenses found]how to fix this  
more detail info https://github.com/community/community/discussions/32030

**Reply by radar-blip** [reply]
@casionone Sorry, I dont know how to fix that issue.
